src/store/auth.js

```diff
@@ -9,6 +10,7 @@ export default {
     verified: false,
     username: '',
     user_id: '',
+    profile_picture_url: '',
```

Comment:
Added as state, since it should be available for the dropdown, and not only for profile pages.